docs/src/main/sphinx/admin/properties-http-client.rst

```diff
@@ -0,0 +1,201 @@
+======================
+HTTP client properties
+======================
+
+HTTP client properties allow you to configure the connection from Trino to
+external services using HTTP.
+
+The following properties can be used after adding the specific prefix to the
+property. For example, for :doc:`/security/oauth2`, you can enable HTTP for
+interactions with the external OAuth 2.0 provider by adding the prefix
+``oauth2-jwk`` to the ``http-client.http.enabled``, and setting the value to
```

Comment:
This isn't a valid property. Let's use `http-client.connect-timeout` and say that we can increase the connection timeout to ten seconds by setting it to `10s`.